universe/package.json

```diff
@@ -26,11 +26,12 @@
     "ray",
     "s3",
     "spark",
-    "tensorflow"
+    "tensorflow",
+    "portworx"
   ],
-  "preInstallNotes": "The Mesosphere Jupyter Service is currently in Beta. There may be bugs, incomplete features, incorrect documentation, breaking changes, or other discrepancies.",
-  "postInstallNotes": "The Mesosphere Jupyter Service has been installed. Please refer to the Service Documentation at https://docs.mesosphere.com/services/beta-jupyter",
-  "postUninstallNotes": "The Mesosphere Jupyter Service has been uninstalled. Thanks for playing. Note: If applicable, any data on the persistent volume on the agent where this service was deployed will be automatically deleted and the volume will be removed.",
+  "preInstallNotes": "This DC/OS Service is currently in preview. There may be bugs, incomplete features, incorrect documentation, or other discrepancies.\n\nDefault password is set to 'jupyter' (unless you install into a folder).\n\nPortworx should be installed on the Private Agents to be able to provision volumes.\n\nAdvanced Installation options notes:\n\nstorage / persistence: create Portworx volume for storing files to survive across restarts or failures. \n\nstorage / portworx_volume_size: define the size of your Portworx persistent volume, e.g. 10GB.\n\nNOTE: If you didn't select persistence in the storage section,\nYOUR DATA WILL NOT BE SAVED IN ANY WAY.\n\nnetworking / port: This DC/OS service can be accessed from any other application through a NAMED VIP in the format service_name.marathon.l4lb.thisdcos.directory:port. Check status of the VIP in the Network tab of the DC/OS Dashboard (Enterprise DC/OS only).\n\nnetworking / external_access: create an entry in Marathon-LB for accessing the service from outside of the cluster\n\nnetworking / external_public_agent_hostname: dns for Marathon-LB, typically set to your public agents public ip.",
```

Comment:
Can you edit the line saying -  `If you didn't select persistence...`? I don't think we give them an option to not select persistence with Portworx.